test/services/katello/pulp/srpm_test.rb

```diff
@@ -33,46 +33,5 @@ def test_update_model_is_idempotent
         assert_equal srpm.reload.updated_at, last_updated
       end
     end
-
-    class RpmTestBase < ActiveSupport::TestCase
-      include RepositorySupport
-
-      def setup
-        User.current = users(:admin)
-
-        @repo = katello_repositories(:fedora_17_x86_64)
-
-        RepositorySupport.create_and_sync_repo(@repo)
-        Katello::Rpm.import_for_repository(@repo)
-        @package_id = @repo.rpms.first.id
-      end
-
-      def teardown
-        RepositorySupport.destroy_repo(@repo)
-        User.current = nil
-      end
-    end
-
-    class RpmTest < RpmTestBase
-      def test_find
-        package = Rpm.find(@package_id)
-
-        refute_nil package
-        refute_empty Katello::Pulp::Rpm.new(package.pulp_id).backend_data
-      end
-
-      def test_requires
-        package = Rpm.find(@package_id)
-        backend_rpm = Katello::Pulp::Rpm.new(package.pulp_id)
-        refute_empty backend_rpm .requires
-        refute_empty backend_rpm .provides
-      end
-
-      def test_ignored_fields
-        refute_includes Katello::Pulp::Rpm::PULP_SELECT_FIELDS, 'changelog'
-        refute_includes Katello::Pulp::Rpm::PULP_SELECT_FIELDS, 'repodata'
-        refute_includes Katello::Pulp::Rpm::PULP_SELECT_FIELDS, 'filelist'
-      end
-    end
   end
```

Comment:
This was duplicated in test/services/katello/pulp/rpm_test.rb and for some reason the tests in the original class were not running on Jenkins. There was a bug in the test https://github.com/Katello/katello/pull/8220/files#diff-20f25fb764ba969e5a677000d85050bbL50 but was never caught cause it wasn't being run. Instead the same tests were being run from here. There is no need to duplicate tests hence removing this and fixing the issue in the original class.